### ISSUE TITLE
action scrip 3 is not in

### ISSUE THREAD

**Issue by solanaea** [ruffle-rs/ruffle#8574]
### Describe the bug

when i play games on this unblocked website many times i get action strip 3 is not there

### Expected behavior

i expected the game to load but it told me there was no action script 3 so i just clicked run anyway and the game was buggy

### Affected platform

Desktop app

### Operating system

chromebook

### Browser

google chrome

**Comment by LukeSaward1** [comment]
What game is it?

**Comment by n0samu** [comment]
Closing in favor of #1368
